lib/ReactViews/Map/Panels/SharePanel/SharePanel.jsx

```diff
@@ -227,18 +234,50 @@ const SharePanel = createReactClass({
         }
     },
 
+    getShareUrlInput(theme) {
+        return (
+            <Input
+                className={Styles.shareUrlfield}
+                light={theme === "light"}
+                dark={theme === "dark"}
+                large
+                type="text" value={this.state.shareUrl}
+                placeholder={this.state.placeholder} readOnly
+                onClick={e => e.target.select()} id='share-url' />
+        );
+    },
+
     renderContent() {
+        if (this.props.catalogShare) {
+            return this.renderContentForCatalogShare();
+        } else {
+            return this.renderContentForPrint();
+        }
+    },
+
+    renderContentForCatalogShare() {
+        return (
+            <Choose>
+                <When condition={this.state.shareUrl === ''}>
+                    <Loader message="Generating share URL..." />
+                </When>
+                <Otherwise>
+                    <div className={Styles.clipboardForCatalogShare}>
+                        <Clipboard theme="light" text={this.state.shareUrl} source={this.getShareUrlInput('light')} id='share-url' />
+                    </div>
+                </Otherwise>
+            </Choose>
+        );
+    },
+
+    renderContentForPrint() {
```

Comment:
Why "for print"?

Reply:
SharePanel still gets used with the print/embed options via the top right icons on the map, "for print" probably wasn't the best name 😃 https://github.com/TerriaJS/terriajs/pull/3278/commits/45d2c3697a184b16f07d9ac88c0627b8f3b67742